pod/perlop.pod

```diff
@@ -1489,13 +1502,14 @@ precedence.  This means that it short-circuits: the right
 expression is evaluated only if the left expression is true.
 
 =head2 Logical or and Exclusive Or
-X<operator, logical, or> X<operator, logical, xor>
-X<operator, logical, exclusive or>
-X<or> X<xor>
+X<operator, logical, or> X<or>
+X<operator, logical, xor> X<operator, logical, exclusive or> X<xor>
+
+There is no low precedence operator for defined-OR.
```

Comment:
This feels slightly better at the end of this section, as an "addendum," in my opinion at least

Reply:
How would that read?  I think the current situation is that it gets lost

Reply:
My feeling is that people would more likely go to this section of the documentation to find out about the or and xor operators, more than to search for a low precedence defined-or, but I have no objections to putting that sentence first either.

Reply:
@Grinnz, I agree that that is why they would come here; but I moved the sentence because I thought that point would be lost where it was, and it's not really a problem to have it moved.

Now you say you don't object to moving the sentence.  What changes need to be made, then?